src/main/java/hu/dungeonhunter/characters/monsters/Goblin.java

```diff
@@ -66,4 +72,14 @@ private void startValues() {
     public int getMonsterDamage() {
         return Dice.rollDice(maxDamage, numOfDices);
     }
+
+    @Override
+    public int getMonsterInitiative(){
+        return getInitiative();
+    }
+
+    @Override
+    public void setMonsterInitiative(int initiative){
+        this.initiative = initiative;
+    }
```

Comment:
This is not needed if you're using @Getter and @Setter on the variable

Reply:
I deleted this lines, thanks!